storage-operator/pkg/controller/volume/volume_controller.go

```diff
@@ -201,7 +201,7 @@ reclaim its storage and remove the finalizers to let the object be deleted.
 
 }}} */
 
-const DEVICE_ANNOTATION = "storage.metalk8s.scality.com/device"
+const DEVICE_ANNOTATION = "storage.metalk8s.scality.com/device-name"
```

Comment:
Now thinking of it... Should this be an annotation, or rather in `status`? :thinking: WDYT @scality/metalk8s

Reply:
IMO, that's what `status` is for, and since we own it... I'd vote for using it. Feels minor however, and we could easily convert between one and the other in the future, if need be.

Reply:
Except that then the UI would need to dispatch between multiple possible locations :)

Reply:
Tie this schema to an `apiVersion`?

Reply:
Done, last commit move things into status.

Reply:
Oh thanks, was not sure if it needed to go in this PR.